test/test.py

```diff
@@ -1,20 +1,23 @@
 import choice
 
+def c():
+  print("Delete confirmed")
+
 # Get a yes or no response (default is no)
 confirm = choice.Binary('Are you sure you want to delete?', False).ask()
 if confirm:
-    deleteIt()
+    c()
```

Comment:
Hey @bastbnl, any reason to put this in a separate `c()` function rather than just inline? Seems to me that just writing `print("Delete confirmed")` here would work too.